### PR TITLE
Replace and add new images

### PR DESCRIPTION
**Pull request by marikaner**
## What Has Changed?

Closes https://github.com/SAP/cloud-sdk-backlog/issues/582

- Added some new images + improved the overview page for JS
- Removed some unused images
- Replaced some existing images

## Manual Checks?

- [ ] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [ ] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [ ] You formatted all changed files with prettier (`npm run prettier`)
- [ ] You tested if the documentation still builds (`npm run build`)
- [ ] You verified all new and changed links still work (changing the `id` or name of a file can break links)
- [ ] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.

### CODE REVIEW

File: static/img/.links.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The images are created using [this PPT presentation](https://sap.sharepoint.com/:p:/r/sites/101252/Shared%20Documents/06_Enablement_Material/SDK-overview.pptx?d=w3b4ab58aac3a465c93189d940fd677c1&csf=1&web=1&e=Pz2o2M).
+
+To export images from there:
+
+1. Mark the parts that you want to export.
+2. Open the right click menu.
+3. Select "Save as Picture...".
```
Comment (FrankEssenberger) [inline static/img/.links.md:7]: minor: there is a vale warning due to the `...`

Comment (FrankEssenberger) [reply static/img/.links.md:7]: From the file name `.links.md` I did not know that it contained the description on `how-to-generate-images` - perhaps you can do something more explicit.

Comment (FrankEssenberger) [reply static/img/.links.md:7]: Also could you either give here and/or in the PPT give some guidance how to get a dark mode version of an normal mode PPT - especially the imagines are not clear to me. Do you have to manually go to the SAP image DB and select the black version or can you invert it in power point?

**Review by FrankEssenberger — CHANGES_REQUESTED** [review]
Content wise fine. I left some comment regarding the image generation

**Review by deekshas8 — CHANGES_REQUESTED** [review]

File: docs/js/overview.mdx [hunk 2]
```diff
@@ -15,27 +15,89 @@ keywords:
...
+
+#### `@sap-cloud-sdk/http-client`
+
+The HTTP client library is a very thin HTTP client, that allows you to execute requests by leveraging the concept of destinations.
+You can flexible configure a request to your needs and send arbitrary HTTP request to the given destination.
+
+#### `@sap-cloud-sdk/odata-v2`, `@sap-cloud-sdk/odata-v4`, and `@sap-cloud-sdk/openapi`
+
+The OData v1, v2 and OpenAPI libraries build the foundation for (pre-)generated client libraries.
+They contain OData and OpenAPI specific abstractions, that the (pre-)generated client libraries use to expose their APIs.
```
[the remark this answers is not in the document]
Comment (marikaner) [reply docs/js/overview.mdx:83]: I meant to express that this applies to pregenerated and generated clients. Maybe just omitting the word altogether would be better?

File: docs/js/overview.mdx [hunk 3]
```diff
@@ -15,27 +15,89 @@ keywords:
...
+  width="100%"
+/>
+<!-- vale on -->
+
+The SAP Cloud SDK consists of two major parts:
+
+- the core libraries
+- the pregenerated client libraries
+
+### The core libraries
```
Comment (marikaner) [inline docs/js/overview.mdx:49]:
```suggestion
### The Core Libraries
```

File: docs/js/overview.mdx [hunk 4]
```diff
@@ -15,27 +15,89 @@ keywords:
...
+<!-- vale on -->
+
+#### `@sap-cloud-sdk/connectivity`
+
+The connectivity library provides functionality to read and exchange with services like the destination, XSUAA and connectivity service.
+It provides the basic types needed to reflect the concept of a [destination](features/connectivity/destination) throughout the SAP Cloud SDK for JavaScript.
+
+#### `@sap-cloud-sdk/http-client`
+
+The HTTP client library is a very thin HTTP client, that allows you to execute requests by leveraging the concept of destinations.
```
Comment (marikaner) [inline docs/js/overview.mdx:77]:
```suggestion
The HTTP client library is a thin HTTP client, that allows you to execute requests by leveraging the concept of destinations.
```

File: docs/js/overview.mdx [hunk 5]
```diff
@@ -15,27 +15,89 @@ keywords:
...
+You can flexible configure a request to your needs and send arbitrary HTTP request to the given destination.
+
+#### `@sap-cloud-sdk/odata-v2`, `@sap-cloud-sdk/odata-v4`, and `@sap-cloud-sdk/openapi`
+
+The OData v1, v2 and OpenAPI libraries build the foundation for (pre-)generated client libraries.
```
Comment (marikaner) [inline docs/js/overview.mdx:82]:
```suggestion
The OData v2, v4, and OpenAPI libraries build the foundation for (pre-)generated client libraries.
```

File: docs/js/overview.mdx [hunk 6]
```diff
@@ -15,27 +15,89 @@ keywords:
...
+
+#### `@sap-cloud-sdk/generator` and `@sap-cloud-sdk/openapi-generator`
 
-## Why the SAP Cloud SDK for JavaScript?
+The OData and OpenAPI generators are CLIs, that generate client libraries based on the services' specifications (either EDMX format or OpenAPI/Swagger).
+You can use them to generate your own clients.
+The pregenerated client libraries are generated using those generators.
 
-The SAP Cloud SDK supports you end-to-end when developing applications that communicate with SAP solutions and services such as SAP S/4HANA Cloud, SAP SuccessFactors, and many others.
+### Pregenerated client libraries
```
Comment (marikaner) [inline docs/js/overview.mdx:91]:
```suggestion
### Pregenerated Client Libraries
```

File: docs/js/overview.mdx [hunk 7]
```diff
@@ -15,27 +15,89 @@ keywords:
...
+
+#### `@sap-cloud-sdk/http-client`
+
+The HTTP client library is a thin HTTP client, that allows you to execute requests by leveraging the concept of destinations.
+You can flexibly configure a request to your needs and send arbitrary HTTP requests to the given destination.
+
+#### `@sap-cloud-sdk/odata-v2`, `@sap-cloud-sdk/odata-v4`, and `@sap-cloud-sdk/openapi`
+
+The OData v2, v4, and OpenAPI libraries build the foundation for (pre-)generated client libraries.
+They contain OData and OpenAPI specific abstractions, that the client libraries (custom and pregenerated) use to expose their APIs.
```
Comment (marikaner) [inline docs/js/overview.mdx:83]:
```suggestion
The OData v2, v4, and OpenAPI libraries build the foundation for client libraries (custom and pregenerated).
They contain OData and OpenAPI specific abstractions, that the client libraries use to expose their APIs.
```

File: static/img/.links.md (hunk 1 above)
Comment (marikaner) [reply static/img/.links.md]: Removed all instructions from here and moved it to the PPT.

**Review by deekshas8 — APPROVED** [review]
With the updated package diagram, LGTM !

**Review by FrankEssenberger — APPROVED** [review]
Like the instruction in the pptx and the new file name.

**Pull request merged**